hacktoberfest_challenges/model_license_other.md

```diff
@@ -0,0 +1,410 @@
+
+# Model `license:other` challenge
+
+## Context
+
+The Hugging Face Hub hosts hundreds of thousands of public models and datasets. Public doesn't necessarily mean open-source without any limitations. Authors can define which license applies to the work they share (e.g. [MIT](https://opensource.org/license/mit/), [Apache2.0](https://www.apache.org/licenses/LICENSE-2.0), [OpenRAIL](https://huggingface.co/blog/open_rail), etc.). All users must be able to quickly know which license applies to which model and even to list models with a specific license (e.g. [Apache2.0](https://huggingface.co/models?license=license:apache-2.0&sort=trending)). The Hub relies on the [Model Card](https://huggingface.co/docs/hub/model-cards) to do so. A Model Card is a file attached to a model providing handy information. They are essential for discoverability, reproducibility and sharing. In our case, we will focus on the [metadata](https://huggingface.co/docs/hub/model-cards#model-card-metadata) section of the Model Card. This metadata contains valuable information, including a `license` tag.
+
+In this challenge, we will focus on models that as the license `"other"`. This means that the model has a custom license defined by the author. For example, LLama2 models have their own license [llama-2-community-license](https://ai.meta.com/llama/license). Historically, this information was not available in the model card metadata. This is now possible by adding the license name and a URL to it:
```

Comment:
Llama 2 license is now supported among the official licenses, so maybe it makes sense to put an unsupported example. WDYT?

Reply:
Oh, didn't know that. Do you have another example that is "quite known"?

Reply:
Maybe coqui is a good example for that, but no strong opinion

Reply:
Maybe coqui is a good example for that, but no strong opinion